examples/RandomNumber/RandomNumber.ino

```diff
@@ -1,5 +1,5 @@
 /*
-  secureElement Random Number
+  SecureElement Random Number
 
   This sketch uses the ECC508/ECC608 or SE050 to generate a random number
   every second and print it to the Serial Monitor
```

Comment:
I suggest adding a note here that the "ConfigurationLocking" example can be used to lock it before using this example.